backend/mora/amqp.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import json
+
+from .settings import (
+    ENABLE_AMQP_MESSAGES,
+    AMQP_MO_EXCHANGE,
+    AMQP_HOST,
+    AMQP_PORT,
+)
+
+
+if ENABLE_AMQP_MESSAGES:
+    import pika
+    conn = pika.BlockingConnection(
```

Comment:
I did it like this because 1) I don't think using `global` is good form and 2) I intended all interactions with `channel` to be made through callables in `amqp.py` which is why I do not want to make it generally available through `flask.g`. The code is very simple like this.

Do you still think I should rewrite this?

Reply:
Please do! I wouldn't worry too much about information hiding unless you _really_ have to; this is Python 😉

Reply:
I am not worried about information hiding. The connection and channel are easily available through the `amqp` module. I'm concerned about having it in the proper namespace and do not see the benefit from complicating the code.

Reply:
Alright, what do you say we classify this as a minor design issue and agree that you or anyone else are welcome to change it as they see fit?